SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKEditorModule.cpp

```diff
@@ -52,24 +49,6 @@ void FSpatialGDKEditorModule::StartupModule()
 	// This is relying on the module loading phase - SpatialGDKServices module should be already loaded
 	FSpatialGDKServicesModule& GDKServices = FModuleManager::GetModuleChecked<FSpatialGDKServicesModule>("SpatialGDKServices");
 	LocalReceptionistProxyServerManager = GDKServices.GetLocalReceptionistProxyServerManager();
-
-	// Allow Spatial Plugin to stop PIE after Automation Manager completes the tests
-	IAutomationControllerModule& AutomationControllerModule =
-		FModuleManager::LoadModuleChecked<IAutomationControllerModule>(TEXT("AutomationController"));
-	IAutomationControllerManagerPtr AutomationController = AutomationControllerModule.GetAutomationController();
-	AutomationController->OnTestsComplete().AddLambda([]() {
-		// Make sure to clear the snapshot in case something happened with Tests (or they weren't ran properly).
-		ASpatialFunctionalTest::ClearAllTakenSnapshots();
-
-#if ENGINE_MINOR_VERSION < 25
-		if (GetDefault<USpatialGDKEditorSettings>()->bStopPIEOnTestingCompleted && GEditor->EditorWorld != nullptr)
-#else
-		if (GetDefault<USpatialGDKEditorSettings>()->bStopPIEOnTestingCompleted && GEditor->IsPlayingSessionInEditor())
-#endif
-		{
-			GEditor->EndPlayMap();
-		}
-	});
```

Comment:
Where is this functionality now? Is there a corresponding TestGym PR?

Reply:
Very sorry about this, realized I forgot to add two files in git.
The functionality is in `SpatialGDKFunctionalTestsModule.cpp` and `SpatialGDKFunctionalTestsModule.h`.